### PR TITLE
feat(param): add option to disable read others functionality

### PR DESCRIPTION
**Pull request by seqizz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,8 @@ paginate = 5
   fullWidthTheme = false
   # center theme with default width
   centerTheme = false
+  # hide "Read more" below posts (text defined below in languages section)
+  disableReadOthers = false
```
Comment (panr) [inline README.md]: I'd change this and make it contrary and commented...

```suggestion
  # disableReadOtherPosts = true
```

File: README.md [hunk 2]
```diff
@@ -84,6 +84,8 @@ paginate = 5
   fullWidthTheme = false
   # center theme with default width
   centerTheme = false
+  # hide "Read more" below posts (text defined below in languages section)
```
Comment (panr) [inline README.md]: I'd change comment:

```suggestion
  # hide "Read Other Posts" below post content (text defined below in languages section)
```

File: layouts/_default/single.html [hunk 3]
```diff
@@ -30,6 +30,7 @@ <h1 class="post-title">
   <div class="post-content">
     {{ .Content }}
   </div>
+  {{ if not .Site.Params.disableReadOthers }}
```
Comment (panr) [inline layouts/_default/single.html]: If this won't work after changes above, I'd change it to:

```suggestion
  {{ if ne .Site.Params.disableReadOtherPosts true }}
```

File: layouts/_default/single.html [hunk 4]
```diff
@@ -30,6 +30,7 @@ <h1 class="post-title">
...
   {{ if or .NextInSection .PrevInSection }}
```
Comment (panr) [inline layouts/_default/single.html]: If you wrap this in another condition, please indent this block for better readability ;-)

**Comment by seqizz** [comment]
Thanks, force pushed :+1:

**Comment by panr** [comment]
Have you test it after changes? Just checking ;-)

**Comment by seqizz** [comment]
Ah sorry, yes re-built with the current one and still working. :+1: 
Btw, will the multiple-content-types branch is going to get merged at some point? :thinking:

**Comment by panr** [comment]
> Btw, will the multiple-content-types branch is going to get merged at some point? 🤔

Good question... I still don't know... No one else asked for this feature...

**Comment by panr** [comment]
@seqizz thank you the feature 🎉 and sorry for a late merge! 😬

**Pull request merged**